.github/workflows/branch-protection.yml

```
@@ -0,0 +1,32 @@
+name: Validate Source Branch
+
+on:
+    pull_request:
+        branches:
+            - main
+
+jobs:
+    check-source-branch:
```

actionlint (1.7.7)

"steps" section is missing in job "check-source-branch"

(syntax-check)